README.md

```diff
@@ -22,15 +22,40 @@ This project consists of a backend server and a React Native app that demonstrat
 - A [React Native Development Environment](https://reactnative.dev/docs/0.70/environment-setup).
 - Latest LTS version of [NodeJS](https://nodejs.org/en/).
 - A Node package manager. Examples in this project will use the latest version of [Yarn v1](https://classic.yarnpkg.com/lang/en/).
+- Create a free tier [Auth0 Account](https://auth0.com/signup?place=header&type=button&text=sign%20up)
+
+## Setting Up Auth0 Authentication
+
+This app implements Auth0 to demonstrate login flow for authenticating and authorizing users
+
+Sign In to your Auth0 account
+
+### Auth0 for the client
+
+Create a new Auth0 `Native` app from the Auth0 user console
+
+- Input your desired `Name`
```

Comment:
There's only one step. We can probably combine this. Also, prefer `enter` over `input`.
```suggestion
Create a new Auth0 `Native` app from the Auth0 user console and enter your desired `Name`
```